src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir-Book";
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+}
+
+.bigFoarm {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.input-container {
+  position: relative;
+}
+
+img {
+  float: left;
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  object-position: center center;
+}
+
+img.small {
+  float: left;
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+  object-position: center center;
+}
+
+input.bigSeachBar {
+  box-sizing: border-box;
+  padding: 24px 62px 23px;
+  width: 100%;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 21.86px;
```

Comment:
use integer values

Reply:
is not fixed.
there are still not integer values at another selector (.small-seachbar)